mentors/delphi/0.json

```
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username": "Neftali",
+    "name": "Germán Estévez -Neftalí-",
+    "link_text": Personal website about Delphi,
```

```suggestion
    "link_text": "Personal website about Delphi",
```
Information needs to be quotes, like you did for `"github_username"` and `"name"`.